packages/components/Notification/src/Notification.styling.ts

```diff
@@ -33,6 +33,12 @@ export const stylingSettings: UseStylingOptions<NotificationProps, NotificationS
       }),
       ['backgroundColor', ...borderStyles.keys, ...layoutStyles.keys],
     ),
+    icon: buildProps(() => ({
+      style: {
+        alignSelf: 'center',
+        marginRight: 16,
```

Comment:
Note for future: I wonder if we want to specify layout tokens for each slots. 
iconMargin, titleMargin, etc. I would say that depends on whether FluentUI Apple specifies those.

Reply:
FUA doesn't but good to keep in mind